lib/solidus_klaviyo/configuration.rb

```diff
@@ -2,7 +2,7 @@
 
 module SolidusKlaviyo
   class Configuration
-    attr_accessor :api_key, :public_key, :default_list, :email
```

Comment:
just a clean up, we don't need the `publick_key` anymore

Reply:
nice!